flake.nix

```diff
@@ -62,8 +71,85 @@
               fourmolu
               haskell-language-server
               hlint
-            ]);
+            ])++ [ cardanoPkgs.cardano-node cardanoPkgs.cardano-cli ] ++ [pkgs.postgresql];
         });
+      testnet = perSystem (system:
```

Comment:
All of the changes here and below (`testnet`, `testnet2`, etc...) are top-level attributes in the flake `outputs`, which is not what you want. Are these going to be added to `packages`? `apps`? You probably eventually want to create an app output that can be called with `nix run` from the command line